web/src/app/modules/views/main/editors/modules/graphical-editor/components/graphical-editor.component.ts

```diff
@@ -303,13 +305,30 @@ export class GraphicalEditor {
                 } else {
                     this.selectedElementService.deselect();
                 }
+
+                const selectedElements = await Promise.all(selections.map(cell => this.dataService.readElement(cell.getId(), true)));
+
+                let cell = undefined;
+                let geo = undefined;
+                if (selectedElements.length === 1) {

```

Comment:
This case is only relevant, if one element is selected. Try this: Do not select a linked node, but start dragging it right away. You'll notice, that you can drag it, even behind the modal background. That's the reason why we need to reset the cell's geometry in case the user presses "Cancel". However, you cannot do this with multiple nodes, because to drag multiple nodes, you need to select them first, which opens the modal. Then, you cannot drag anymore.